### PR TITLE
feat: block type

### LINKED ISSUE #3: Create block type

### PR DESCRIPTION
**Pull request by baribari2**
Adds the block type, proto conversion, and map function for `subscribe_new_blocks`. Resolves #3

### CODE REVIEW

File: fiber/client.py [hunk 1]
```diff
@@ -47,6 +47,45 @@ def proto_to_tx(proto: eth_pb2.Transaction):
     tx.s = encode_hex(proto.s)
     return tx
 
+class Block:
+    hash: str
+    number: int
+    parent_hash: str
+    timestamp: int
+    producer: str
```
Comment (jonasbostoen) [inline fiber/client.py:55]: This key doesn't exist

File: fiber/client.py [hunk 2]
```diff
@@ -47,6 +47,45 @@ def proto_to_tx(proto: eth_pb2.Transaction):
...
+    base_fee_per_gas: int
+    extra_data: str
+    fee_recipient: str
+    gas_limit: int
+    gas_used: int
+    logs_bloom: bytes
+    prev_randao: bytes
+    receipt_root: bytes
+    state_root: bytes
+    transactions: list[Transaction]
```
Comment (jonasbostoen) [inline fiber/client.py:65]: I think for transactions, it gets a bit expensive if we loop and convert every single one of them. I think we should have another method `subscribe_new_headers` that does the same as `subscribe_new_blocks` but without the conversion. Wdyt?

**Review by jonasbostoen — COMMENTED** [review]
Ty for this! A couple comments

File: fiber/client.py (hunk 2 above)
Comment (baribari2) [reply fiber/client.py:65]: I think that makes sense!

**Comment by jonasbostoen** [comment]
Thanks a lot sir

**Pull request merged**